### PR TITLE
Add support for Paper 1.21+ to AnvilInput

### PR DESCRIPTION
**Pull request by mfnalex**

### CODE REVIEW

File: inventory-framework-anvil-input/src/main/java/me/devnatan/inventoryframework/AnvilInputNMS.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ class AnvilInputNMS {
                     getMethod(ENTITY_PLAYER, "nextContainerCounter", MethodType.methodType(int.class));
 
             GET_PLAYER_INVENTORY =
-                    getMethod(ENTITY_PLAYER, "fN", MethodType.methodType(playerInventoryClass), false, "fR");
+                    Objects.requireNonNull(getMethod(ENTITY_PLAYER, "fN", MethodType.methodType(playerInventoryClass), false, "fR", "getInventory"));
```
Comment (nicolube) [inline inventory-framework-anvil-input/src/main/java/me/devnatan/inventoryframework/AnvilInputNMS.java:69]: Objects.requireNonNull is not nessery

File: inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/McVersion.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+import org.bukkit.Bukkit;
+
+import java.util.Objects;
+
+/**
+ * Represents a Minecraft version.
+ *
+ * @author mfnalex
+ */
```
Comment (nicolube) [inline inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/McVersion.java:11]: Other classes do not have headers like this.

File: inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/ReflectionUtils.java [hunk 3]
```diff
@@ -87,9 +85,14 @@ public final class ReflectionUtils {
...
         }
-        if (found == null)
-            throw new IllegalArgumentException(
-                    "Failed to parse server version. Could not find any package starting with name: 'org.bukkit.craftbukkit.v'");
+        if (found == null) {
+			try {
+				Class.forName("org.bukkit.craftbukkit.entity.CraftPlayer");
+			} catch (ClassNotFoundException e) {
+				throw new IllegalArgumentException(
+					"Failed to parse server version. 6Could not find any package starting with name: 'org.bukkit.craftbukkit.v'");
```
Comment (nicolube) [inline inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/ReflectionUtils.java:93]: Spelling

File: inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/ReflectionUtils.java [hunk 4]
```diff
@@ -182,9 +153,9 @@ public static String getVersionInformation() {
     private static final MethodHandle SEND_PACKET;
 
     static {
-        Class<?> entityPlayer = getNMSClass("server.level", "EntityPlayer");
-        Class<?> craftPlayer = getCraftClass("entity.CraftPlayer");
-        Class<?> playerConnection = getNMSClass("server.network", "PlayerConnection");
+        Class<?> entityPlayer = Objects.requireNonNull(getNMSClass("server.level", "EntityPlayer"));
```
Comment (nicolube) [inline inventory-framework-platform-bukkit/src/main/java/me/devnatan/inventoryframework/runtime/thirdparty/ReflectionUtils.java:156]: Well, Objects.requireNonNull is still not quite needed here.

Existing code should remain unchanged if not necessary to Implement a feature

**Review by nicolube — COMMENTED** [review]
Wich MC Versions has this update been tested for?

**Comment by mfnalex** [comment]
> Wich MC Versions has this update been tested for?

1.21.1 and 1.20.2

**Pull request merged**